creme/settings.py

```diff
@@ -896,6 +897,9 @@
 
     'creme_core/js/widgets/list/pager.js',
 
+    'creme_core/js/widgets/form/chosen.js',
```

Comment:
Can now add a comment like "# DEPRECATED: to be removed in Creme 2.5" please, so I won't forget it.